### PR TITLE
{bio}[GCCcore-10.3.0] minimap2 bumped to 2.20

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/1447048ff542e10446432f396dc7c21e for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/m/minimap2/minimap2-2.20-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    'LICENSE.txt', 'NEWS.md', 'README.md',
+    (['%(name)s.1'], 'share/man/man1')
+]
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s', 'lib/lib%(name)s.a'],
+    'dirs': ['include']
+}
+
+sanity_check_commands = ["minimap2 --help"]
```
Comment (verdurin) [inline easybuild/easyconfigs/m/minimap2/minimap2-2.20-GCCcore-10.3.0.eb:44]: @sassy-crick any reason why you've removed part of the check here?

Comment (sassy-crick) [reply easybuild/easyconfigs/m/minimap2/minimap2-2.20-GCCcore-10.3.0.eb]: No. That was a mistake. Sorry for this, I have corrected that.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node379.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/807fab4e789744d46844bd1538ca0b0d for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node115.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/37bf983b69a9bae0b68efbb83963233e for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @sassy-crick !

**Pull request merged**